include/realizations/catchment/Bmi_Multi_Formulation.hpp

```diff
@@ -410,7 +410,7 @@ namespace realization {
                 LOG(throw_msg, LogLevel::WARNING);
                 throw std::runtime_error(throw_msg);
             }
-            return availableData[output_name]->get_value(CatchmentAggrDataSelector(this->get_catchment_id(),output_name, init_time, duration_s, output_units), m);
+            return availableData[output_name]->get_value(CatchmentAggrDataSelector(this->get_catchment_id(),output_name, init_time, duration_s, output_units, 0), m);
```

Comment:
Why is this not just
```suggestion
            return availableData[output_name]->get_value(selector, m);
```

Reply:
Same for `get_values` just below